diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/AvoidUtilityClassWarnTest.kt

```diff
@@ -79,4 +79,21 @@ class AvoidUtilityClassWarnTest: LintTestBase(::AvoidUtilityClass) {
                 LintError(2,1, ruleId, "${AVOID_USING_UTILITY_CLASS.warnText()} StringUtils")
         )
     }
+
+    @Test
+    @Tag(WarningNames.AVOID_USING_UTILITY_CLASS)
+    fun `test with class without identifier`() {
+        lintMethod(
+                """
+                    fun foo() {
+                    window.addMouseListener(object : MouseAdapter() {
+                        override fun mouseClicked(e: MouseEvent) { /*...*/ }
+                    
+                        override fun mouseEntered(e: MouseEvent) { /*...*/ }
+                    })
+                    }
+                """.trimMargin(),
+                LintError(2,45, ruleId, "${AVOID_USING_UTILITY_CLASS.warnText()} object : MouseAdapter() {...")
```

Comment:
This warning is incorrect - this is simply a class usage, what if it is a library class that can't be changed? I think you should only check named classes and objects.